sdk/storage/azure-storage-queue/src/test/java/com/azure/storage/queue/spock/QueueTestHelper.groovy

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.queue.spock
+
+import com.azure.core.http.rest.Response
+import com.azure.core.implementation.util.ImplUtils
+import com.azure.core.util.configuration.ConfigurationManager
+import com.azure.storage.queue.models.CorsRule
+import com.azure.storage.queue.models.Logging
+import com.azure.storage.queue.models.Metrics
+import com.azure.storage.queue.models.QueueItem
+import com.azure.storage.queue.models.RetentionPolicy
+import com.azure.storage.queue.models.SignedIdentifier
+import com.azure.storage.queue.models.StorageErrorCode
+import com.azure.storage.queue.models.StorageErrorException
+import com.azure.storage.queue.models.StorageServiceProperties
+
+import java.time.Duration
+
+import static org.junit.Assert.assertEquals
+import static org.junit.Assert.assertNull
+import static org.junit.Assert.assertTrue
+
+class QueueTestHelper {
```

Comment:
A lot of this class should be updated to return booleans instead of running asserts given how Spock and JUnit differ. Use them as boolean flags in `then` and `expect` sections.

Reply:
Will do a double check and clean up